src/content/reference/web-interface/app-catalog.md

```diff
@@ -0,0 +1,43 @@
++++
+title = "The Giant Swarm App Catalog in The Web Interface"
+description = "What the Giant Swarm App Catalog looks like on our Web Interface and how to use it."
+date = "2019-07-05"
+layout = "subsection"
+weight = 10
++++
+
+# The Giant Swarm App Catalog in the Web Interface
+
+This page will give you an overview of how to do some common tasks related to the
+Giant Swarm App Catalog using our Web Interface.
+
+Go here for an [overview of the Giant Swarm App Catalog](/basics/app-catalog/) instead.
+
+### Viewing all App Catalogs
+
+Our web interface lets you browse the App Catalogs installed on your Control Plane.
+Click on "App Catalogs" in the navigation menu. The "App Catalogs" link will only
```

Comment:
there's also the link in cluster details page

Reply:
True but, I think there's no need to mention that here as well. Users will find it.